### ISSUE TITLE
Unable to create a PR from https://qchateau.github.io/conan-center-bot/#/updatable

### ISSUE THREAD

**Issue by EstebanDugueperoux2** [qchateau/conan-center-bot#111]
Hi,

I'm unable to create a PR from https://qchateau.github.io/conan-center-bot/#/updatable.
Trying to create a PR for [libde265](https://github.com/strukturag/libde265) I get a "Pull request creation failed. Validation failed: must be a collaborator" error message.
See also comments from https://github.com/conan-io/conan-center-index/pull/18603#issuecomment-1640039576.

Regards.

**Comment by jtbandes** [comment]
I'm hitting the same error trying to create a PR for mcap: https://github.com/conan-io/conan-center-index/compare/conan-io:master...qchateau:ccb-mcap-1.2.0?expand=1

**Comment by jtbandes** [comment]
FWIW, cc @ericLemanissier - I get the same error when trying to use your fork as well. https://github.com/conan-io/conan-center-index/compare/conan-io:master...ericLemanissier:ccb-mcap-1.2.0?expand=1
<img width="502" alt="image" src="https://github.com/qchateau/conan-center-bot/assets/14237/b4e8cfae-6ab6-45fb-a2e7-ab16d1855e55">

**Comment by ericLemanissier** [comment]
OK. It must be that either CCI or github policy is refusing pull requests if the submitter is not contributor to the source repository.

**Comment by qchateau** [comment]
Yes it still works for me. I tried to change a few settings, can you retry from mine ?
Else, try to fork my fork (...) and open your PR from there

**Comment by ericLemanissier** [comment]
Same error. Maybe it's because of `Allowing edits by maintainers` which is ticked by default?

**Comment by jtbandes** [comment]
I still get the same error too. It seems like it could be a GitHub policy. Would it make sense to enable the bot itself to submit a PR (perhaps only upon request)?

**Comment by jtbandes** [comment]
Do you think it's an issue with the upstream repo or with the fork? Maybe we should reach out to GitHub support to understand what the error really means by "must be a collaborator"?

**Comment by ericLemanissier** [comment]
This should be raised to github support yes. It happens when the person submitting the PR is not a collaborator to the fork being pulled, so the issue could originate in both the fork or the upstream.

**Comment by jtbandes** [comment]
I've opened a discussion here: https://github.com/orgs/community/discussions/61472

**Comment by Dobiasd** [comment]
And updates on this?
